index.html

```diff
@@ -2056,8 +2056,9 @@ <h3>capabilityInvocation</h3>
     "publicKeyBase58": "H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV"
     }
   ],
+  <span class="comment">...</span>
 }
-<span class="comment">...</span>
+
```

Comment:
Assume this extra spacing wasn't intended?

Reply:
Yup, forgot to remove it when I moved the line. Thanks.